vendor/composer/platform_check.php

```diff
@@ -4,8 +4,8 @@
 
 $issues = array();
 
-if (!(PHP_VERSION_ID >= 50600)) {
-    $issues[] = 'Your Composer dependencies require a PHP version ">= 5.6.0". You are running ' . PHP_VERSION . '.';
+if (!(PHP_VERSION_ID >= 70200)) {
```

Comment:
You should launch composer with --ignore-platform-reqs
Could you please check that shoppingfeed/php-feed-generator ~1.4.0 remain compatible with PHP 5.6 ?

Reply:
shoppingfeed/php-feed-generator must be compatible with PHP 7.1